text2sql-backend/pyproject.toml

```diff
@@ -11,21 +11,15 @@ python = ">=3.11,<3.12"
 fastapi = "0.100.0"
 uvicorn = {extras = ["standard"], version = "^0.21.1"}
 llama-index = "^0.9.42.post2"
-sqlparse = "^0.4.3"
-psycopg2 = "^2.9.6"
+psycopg2-binary = "^2.9.9"
```

Comment:
Are you sure using the binary version here is ok? Don't personally know what the difference is, but I'd rather stick to regular psycopg2

Reply:
It should be fine, it's just easier to install as a binary, less headachy.

Author is the same https://pypi.org/user/piro/